doc/release-notes/master.txt

```diff
@@ -51,6 +51,9 @@ Sprokit
 
 Sprokit: Processes
 
+ * Added an extension plugin capability to embedded_pipeline that
```

Comment:
The extension plugin is implemented in the embedded pipeline class. It would need to be implemented in pipeline runner to be more generally available.